src/main/resources/db/changelog/changes/08-update-users-table.yaml

```diff
@@ -0,0 +1,20 @@
+databaseChangeLog:
+  - changeSet:
+      id: 8
+      author: ChorniyPistolet
+      preConditions:
+        - onFail: CONTINUE
+          sqlCheck:
+            expectedResult: 0
+            sql: SELECT COUNT(*) FROM users_roles WHERE role_id IS NULL
+      changes:
+        - sql:
+            dbms: mysql
+            endDelimiter: ";"
+            splitStatements: true
+            stripComments: true
+            sql: |
+              INSERT INTO users_roles (user_id, role_id)
+              SELECT id, 1
+              FROM users
+              WHERE id NOT IN (SELECT user_id FROM users_roles)
```

Comment:
```suggestion
            sql: |
              INSERT INTO users_roles (user_id, role_id)
              SELECT id, 1
              FROM users
              WHERE id NOT IN (SELECT user_id FROM users_roles)

```
![image](https://github.com/user-attachments/assets/8d497a9d-3664-43f2-991b-36281c7d2002)
go through the files in your PR, add empty line where needed